### PR TITLE
uui-color-swatches: color swatches changes form when showLabel is true

### PR DESCRIPTION
**Pull request by loivsen**
Color swatches changes form when  showLabel is set to true.
Added logic for overflow on swatches labels
Color swatches are now an actual button
Removed extra padding for swatches
Added padding for color picker's swatch area

### CODE REVIEW

File: packages/uui-color-picker/lib/uui-color-picker.element.ts
Comment (nielslyngsoe) [inline packages/uui-color-picker/lib/uui-color-picker.element.ts:643]: An ID most only be present once pr. document fragment(web component, shadow dom). So we should be using a class here, as there is multiple swatch elements.

File: packages/uui-color-swatch/lib/uui-color-swatch.element.ts
Comment (nielslyngsoe) [inline packages/uui-color-swatch/lib/uui-color-swatch.element.ts:174]: since width is already set, I would think its unnessecary to add the max-width. Sorry If I'm mistaken.

File: packages/uui-color-swatch/lib/uui-color-swatch.element.ts
Comment (nielslyngsoe) [inline packages/uui-color-swatch/lib/uui-color-swatch.element.ts:189]: Really good, thanks for being aware on these details.

File: packages/uui-color-swatch/lib/uui-color-swatch.element.ts
Comment (nielslyngsoe) [inline packages/uui-color-swatch/lib/uui-color-swatch.element.ts:255]: If I'm not mistaken its unnecessary complex to only add the title if its overflowing.
I think we should always add the title, Both to keep it simpler and for the purpose of accessibility. :-)

Comment (nielslyngsoe) [reply packages/uui-color-swatch/lib/uui-color-swatch.element.ts:255]: BTW. This comment is regarding the query and the `_detectOverflow ` method.

Comment (nielslyngsoe) [reply packages/uui-color-swatch/lib/uui-color-swatch.element.ts:255]: I might have to mention it. I think we can just put the label on the button, similar to aria-label. And always have it there. So I think aria-label on the button can just be renamed to label.

**Review by nielslyngsoe — COMMENTED** [review]
Hi Lone, I have added a few comments here and there. I hope they make sense, Thanks

File: packages/uui-color-swatch/lib/uui-color-swatch.element.ts
Comment (loivsen) [reply packages/uui-color-swatch/lib/uui-color-swatch.element.ts:174]: max-width is important for the overflow, but seems like removing width is fine :)

File: packages/uui-color-picker/lib/uui-color-picker.element.ts
Comment (loivsen) [reply packages/uui-color-picker/lib/uui-color-picker.element.ts:643]: there is only one swatches element. It is the swatch element there is multiple of 😉

File: packages/uui-color-swatch/lib/uui-color-swatch.element.ts
Comment (loivsen) [reply packages/uui-color-swatch/lib/uui-color-swatch.element.ts]: Im not entirely sure i get the last part - but I removed the overflow detection and added the title on the button now

Comment (nielslyngsoe) [reply packages/uui-color-swatch/lib/uui-color-swatch.element.ts]: perfect, that was it :-)

File: packages/uui-color-picker/lib/uui-color-picker.element.ts
Comment (nielslyngsoe) [reply packages/uui-color-picker/lib/uui-color-picker.element.ts:643]: sorry my mistake. Thanks

File: packages/uui-color-swatch/lib/uui-color-swatch.element.ts
Comment (nielslyngsoe) [reply packages/uui-color-swatch/lib/uui-color-swatch.element.ts:172]: Good :-)

**Review by nielslyngsoe — APPROVED** [review]

**Comment by nielslyngsoe** [comment]
@loivsen is seems there is a test failing. Will you have a look at it?.. I have approved it, so once it goes through it should be merged. :-)

**Pull request merged**
